src/schemes/solid/discrete_element_method/rhs.jl

```diff
@@ -22,93 +12,87 @@ function interact!(dv, v_particle_system, u_particle_system, v_neighbor_system,
                                                                             neighbor,
                                                                             pos_diff,
                                                                             distance
-        m_a = particle_system.mass[particle]
+        distance < sqrt(eps()) && return
 
+        # Retrieve particle properties.
```

Comment:
```suggestion
        # Retrieve particle properties
```
Please check all comments.